src/tests/Loader/classloader/RefFields/InvalidCSharp.il

```diff
@@ -107,15 +107,15 @@
         ldarg.0
         ldarga.s 1
         mkrefany !T
-        stfld typedref InvalidCSharp.WithTypedReferenceField`1::Field
+        stfld typedref class InvalidCSharp.WithTypedReferenceField`1<!T>::Field
```

Comment:
I don't understand why this works on CoreCLR. It's loading a field on an uninstantiated type.